sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
Just to be clear @darshan-serpent  I agree that the readme is correct after your changes now. However, the correct behavior is to change the requested dates of all the lines if the user modifies the requested date in the sale order.